docs/config.toml

```diff
@@ -221,7 +221,11 @@ defaultContentLanguage = "en"
     identifier = "wiring"
     weight = 303
     parent = "authoring-bundles"
-
+[[menu.main]]
```

Comment:
oh my gosh! I hope you didn't think that I was trying to say that you should write net new docs! I mean that's great that you did, I was just saying that you should link to the autogenerated docs for `porter explain` from the nav. Sorry about that if I was unclear. No worries, we can add the link in a follow-up.

![Screen Shot 2019-09-26 at 2 17 14 PM](https://user-images.githubusercontent.com/1368985/65718057-5b084500-e068-11e9-8eb4-cde11348a61e.png)

Reply:
oh i thought i added that one too :) sorrrry.